packages/appstore/src/components/cfds-listing/index.tsx

```diff
@@ -140,14 +140,22 @@ const CFDsListing = () => {
                         const has_mt5_account_status = existing_account.status
                             ? getMT5AccountAuthStatus(existing_account.status)
                             : null;
+                        const is_demo_account = existing_account.account_type === 'demo';
+                        const is_demo_get_account = is_demo && existing_account.action_type === 'get';
                         return (
                             <TradingAppCard
                                 action_type={existing_account.action_type}
                                 availability={selected_region}
                                 clickable_icon
                                 icon={existing_account.icon}
-                                sub_title={existing_account?.sub_title}
-                                name={!has_mt5_account_status ? existing_account?.name : ''}
+                                sub_title={localize('{{sub_title}}{{demo}}', {
+                                    sub_title: existing_account?.sub_title,
+                                    demo: is_demo_account ? ' Demo' : '',
+                                })}
+                                name={localize('{{name}}{{demo}}', {
```

Comment:
```suggestion
                                name={localize('{{name}}{{demo}}', {
```
Same here wont' work

Reply:
Done